README.md

```diff
@@ -8,7 +8,7 @@ Please refer to the sections below for more detailed information. The instructio
 
 1. [Download](https://developer.apple.com/downloads/index.action) and install Xcode. Refer to the [.xcode-version](./.xcode-version) file for the minimum required version.
 1. [Clone this repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) in the folder of your preference.
-1. Run `cd WordPress; make dependencies; cd ..`
+1. Run `cd WordPress; rake dependencies; cd ..`
```

Comment:
Given [the `Rakefile` is in the root of the project](https://github.com/wordpress-mobile/WordPress-iOS/blob/d28ffeb80069d1ac689c1442f2b52d62c888ccb9/Rakefile):

```suggestion
1. Run `rake dependencies`
```